views/benefits/ei_sickness_cerb-en.njk

```diff
@@ -16,8 +16,8 @@
       <li> Did not leave your job voluntarily. </li>
       <li>Earned at least $5,000 in 2019 or the last 12 months.</li>
       <li>Stopped working because of COVID-19: <ul class='list-disc list-outside'>
-          <li>In the first 4 weeks you apply for, you received no income for at least 14 consecutive days.</li>
-          <li>For the next 4-week periods after your first application, you expect to receive no income at all for the whole period.</li>
+          <li>In the first 4 weeks you apply for, you received less than $1,000 for at least 14 consecutive days.    </li>
```

Comment:
this might just be github, but there's extra space or a tab at the end here

Reply:
I'll remove it, it's a copy paste error